examples/virtual-networks/private-link-service/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "resource_group_name" {
+  description = "The name of the resource group the Private Link Service is located in."
+  default = ["example-private-link-service"]
```

Comment:
Is there a reason this is an array not a string?

Reply:
...because that is the way the API is written :/

Reply:
same question